Source/DafnyCore/AST/TokenRangeOrigin.cs

```diff
@@ -4,6 +4,7 @@
 namespace Microsoft.Dafny;
 
 [method: SyntaxConstructor]
+[SyntaxBaseType(typeof(IOrigin))]
```

Comment:
nit: Isn't this redundant?

Reply:
No because IOrigin is an interface, so it's not a Type.BaseType